src/hoc/selectable-enhance.js

```diff
@@ -76,12 +76,32 @@ export const SelectableContainerEnhance = (Component) => {
           key: this._keyIndex,
           style: mergedChildrenStyles,
           nestedItems: child.props.nestedItems.map((child) => this._extendChild(child, styles, selectedItemStyle)),
+          initiallyOpen: this._isInitiallyOpen(child),
         });
       } else {
         return child;
       }
     },
 
+    _isInitiallyOpen(child) {
```

Comment:
What about the `_` convention?
I think that all the methods are private, I don't see the need for it.